### PR TITLE
add PTP images

### PR DESCRIPTION
**Pull request by zenghui-shi**

### CODE REVIEW

**Comment by zshi-redhat** [comment]
OCP image builds were added via:
ptp-operator: https://jira.coreos.com/browse/ART-1090
linuxptp-daemon: https://jira.coreos.com/browse/ART-1093

**Comment by zshi-redhat** [comment]
Are job suppose to fail on release-4.4 branches?

**Comment by smarterclayton** [comment]
Do you already have OCP images created and building?

File: ci-operator/config/openshift/linuxptp-daemon/openshift-linuxptp-daemon-master.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+base_images:
+  base:
+    name: "4.3"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.10
```
Comment (smarterclayton) [inline ci-operator/config/openshift/linuxptp-daemon/openshift-linuxptp-daemon-master.yaml]: You should be on golang 1.12 or later

**Comment by zshi-redhat** [comment]
> Do you already have OCP images created and building?

Yes, OCP image builds were added via:
ptp-operator: https://jira.coreos.com/browse/ART-1090
linuxptp-daemon: https://jira.coreos.com/browse/ART-1093

File: ci-operator/config/openshift/linuxptp-daemon/openshift-linuxptp-daemon-master.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+images:
+- dockerfile_path: Dockerfile.rhel7
+  from: base
+  to: linuxptp-daemon
```
Comment (smarterclayton) [inline ci-operator/config/openshift/linuxptp-daemon/openshift-linuxptp-daemon-master.yaml]: The name of this image should be `ptp-daemon` or just `ptp`.  Will there ever be other images?

Comment (zshi-redhat) [reply ci-operator/config/openshift/linuxptp-daemon/openshift-linuxptp-daemon-master.yaml]: there are two images in the PR
one is `linuxptp-daemon` which runs as a kube DaemonSet
the other one is for ptp operator, image called `ptp-operator`, which will be a redhat operator that can be installed via redhat operator src.

Comment (zshi-redhat) [reply ci-operator/config/openshift/linuxptp-daemon/openshift-linuxptp-daemon-master.yaml]: `linuxptp` is one of the implementations of PTP(Precision Time Protocol).
`Linuxptp` rpm package is supported on RHEL.

Comment (zshi-redhat) [reply ci-operator/config/openshift/linuxptp-daemon/openshift-linuxptp-daemon-master.yaml]: @smarterclayton I can change the image name from `linuxptp-daemon` to `linuxptp` for origin build, shall I also change the OCP image name to the same?

File: ci-operator/config/openshift/linuxptp-daemon/openshift-linuxptp-daemon-master.yaml (hunk 1 above)
Comment (zshi-redhat) [reply ci-operator/config/openshift/linuxptp-daemon/openshift-linuxptp-daemon-master.yaml]: Done

File: ci-operator/config/openshift/linuxptp-daemon/openshift-linuxptp-daemon-master.yaml (hunk 2 above)
Comment (zshi-redhat) [reply ci-operator/config/openshift/linuxptp-daemon/openshift-linuxptp-daemon-master.yaml]: Created ticket to change the image name of OCP: https://jira.coreos.com/browse/ART-1205

Comment (zshi-redhat) [reply ci-operator/config/openshift/linuxptp-daemon/openshift-linuxptp-daemon-master.yaml]: @smarterclayton hi, above ticket has been resolved and marked as Done (OCP image name was changed to `ptp`). `image-references` in `ptp-operator` has been updated to use the right image name `origin-ptp`. I think we should be good to go with one? please let me know if anything else is missing. Thanks!

**Pull request merged**